requests/models.py

```diff
@@ -869,5 +869,5 @@ def close(self):
         """
         if not self._content_consumed:
             self.raw.close()
-
-        return self.raw.release_conn()
+        release_conn = getattr(self.raw, 'release_conn', lambda *args: None)
```

Comment:
I'd rather do an `if` here than create this temporary lambda every single time. =)